### ISSUE TITLE
Question - Does this also register the right click option in VS?

### ISSUE THREAD

**Issue by bbakermmc** [prafullbhosale/CustomAspNetCodeGeneratorTemplate#2]
Looking to implement some custom templates myself and I came across your repository.  Does this also register the items to be seen in VS so you dont need to run CLI to make the template.  The way Microsoft.VisualStudio.Web.CodeGenerators.Mvc puts the options in the scaffolding option.

![image](https://user-images.githubusercontent.com/30090055/36747780-ea1d0174-1bc3-11e8-95ff-b01f3afcf289.png)

**Comment by prafullbhosale** [comment]
No, this is for command line only. The Visual Studio UI is built separately for the scaffolders from CodeGenerators.Mvc package. Currently VS tooling for scaffolding doesn't have the capability to detect scaffolders installed via NuGet packages and show up a UI based on that. 

The only way to provide an experience via VS right now would be to create an extension for VS, which does similar things. 

Unfortunately, I don't have a sample for that yet.
